OracleGoldenGate/setupLinuxEnv.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2016 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: December, 2016
+# Author: gerald.venzl@oracle.com
+# Description: Sets up the unix environment for DB installation.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /opt/oracle as user 'oracle' to proceed with Oracle installation
+# ------------------------------------------------------------
+
+# Setup environment for GoldenGate.   
+mkdir -p $OGG_INSTALL_DIR && \
+chown -R oracle:oinstall $INSTALL_DIR && \
+chown -R oracle:oinstall $OGG_INSTALL_DIR && \
+chmod ug+x $INSTALL_DIR && \
+chmod ug+x $OGG_INSTALL_DIR && \
+yum -y -q install vim-enhanced && \
```

Comment:
Actually it is something a user would do in a running container such as  Oracle database container where they can set up the GoldenGate configuration while the database is running.   The configuration through GoldenGate parameter files  is the "development" process used to build a replication data flow configuration and the built-in editor functions as the IDE.   This allows the image to be generic, but having the ability to spawn multiple containers to represent a GoldenGate pattern.  So for example there could be a source container with GoldenGate + the Oracle database and a target container with GoldenGate + the Oracle database.     Or there could be GoldenGate running on an Oracle database container streaming data to something like the WebLogic JMS container or to a Confluent Kafka container.     Basically, you have to think of it like a development environment.   

Now I can certainly access the GoldenGate configuration externally using the hosts editor or the GoldenGate Studio UI, or I can put these tools in their own image and have them run under a container.    For now I'm just starting with little steps and still need the editor within the container.   I get it that this adds extract "weight" to the container and the intent is to be be simple and light.